src/main/scala/io/sdkman/changelogs/JMeterMigration.scala

```diff
@@ -2,21 +2,41 @@ package io.sdkman.changelogs
 
 import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
 import com.mongodb.client.MongoDatabase
+import org.bson.Document
 
 @ChangeLog(order = "051")
 class JMeterMigration {
+  val CandidateName  = "jmeter"
+  val DefaultVersion = "5.4.3"
```

Comment:
You can probably ditch these constants, as they will only ever be used in one place and make things more confusing to grok.

Reply:
Removed